### PR TITLE
docs: add documentation about resource placement

### PR DESCRIPTION
**Pull request by makkes**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-74752

## Description of changes being made
add documentation about resource placement

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Comment by makkes** [comment]
@sonali-mishra-22 fyi

File: pages/dkp/kommander/1.4/projects/index.md [hunk 1]
```diff
@@ -69,3 +69,58 @@ The following procedures are supported for projects:
 - [Manage Project Quotas](/dkp/kommander/1.4/projects/project-quotas-limit-range)
 - [Manage Project Roles](/dkp/kommander/1.4/projects/project-roles)
 - [Manage Project Secrets](/dkp/kommander/1.4/projects/project-secrets)
+
+## Federation to select clusters
+
+Sometimes it is desirable to narrow down the list of clusters receiving a resources such as a service or a quota to a subset of clusters in a particular Project. Use the `spec.placement` field of federated resources in this case. The following two examples show how this is accomplished using either a list of clusters or a list of labels that a particular cluster has to match.
```
Comment (armandgrillet) [inline pages/dkp/kommander/1.4/projects/index.md:75]:
```suggestion
Sometimes it is desirable to narrow down the list of clusters receiving a resource such as a service or a quota to a subset of clusters in a particular Project. Use the federated resources' field `spec.placement` in this case. The following two examples show how this is accomplished, using either a list of clusters or a list of labels that a particular cluster has to match.
```

File: pages/dkp/kommander/1.4/projects/index.md [hunk 2]
```diff
@@ -69,3 +69,58 @@ The following procedures are supported for projects:
 - [Manage Project Quotas](/dkp/kommander/1.4/projects/project-quotas-limit-range)
 - [Manage Project Roles](/dkp/kommander/1.4/projects/project-roles)
 - [Manage Project Secrets](/dkp/kommander/1.4/projects/project-secrets)
+
+## Federation to select clusters
```
Comment (armandgrillet) [inline pages/dkp/kommander/1.4/projects/index.md:73]:
```suggestion
## Federation on selected clusters
```

File: pages/dkp/kommander/1.4/projects/index.md [hunk 3]
```diff
@@ -69,3 +69,58 @@ The following procedures are supported for projects:
...
+
+The labels provided in the `matchLabels` field are matched against the `KubefedCluster` resource on the Kommander cluster. Run
+
+```sh
+kubectl get kubefedclusters -A
+```
+
+to find a list of all attached clusters and set the labels on each targeted cluster accordingly.
+
+More info on the placement of federated resources is found in the [kubefed user documentation](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#using-cluster-selector).
```
Comment (armandgrillet) [inline pages/dkp/kommander/1.4/projects/index.md:126]:
```suggestion
More information on the placement of federated resources is available in the [kubefed user documentation](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#using-cluster-selector).
```

**Review by cmastr — APPROVED** [review]

**Review by JohnB64-JBV — APPROVED** [review]

**Review by JohnB64-JBV — CHANGES_REQUESTED** [review]
Armand's suggestions look like good changes to make.

File: pages/dkp/kommander/1.4/projects/index.md [hunk 4]
```diff
@@ -59,13 +63,58 @@ spec:
       matchLabels:
         cluster: prod
 ```
+# Federation to select clusters
 
-The following procedures are supported for projects:
+You may want to narrow down the list of clusters receiving resources, such as a service or a quota to a subset of clusters in a particular Project. Use the federated resources `spec.placement` field in this case. The following two examples show how set up either a list of clusters or a list of labels that a particular cluster must match.
```
Comment (cneth) [inline pages/dkp/kommander/1.4/projects/index.md:68]:
```suggestion
You may want to limit the list of clusters receiving resources, such as a service or a quota to a subset of clusters in a particular Project. Use the federated resources `spec.placement` field in this case. The following two examples show how set up either a list of clusters or a list of labels that a particular cluster must match.
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Comment by makkes** [comment]
@armandgrillet @JohnB64-JBV @cneth I incorporated all your feedback. Please have another look.

File: pages/dkp/kommander/1.4/projects/index.md [hunk 5]
```diff
@@ -9,40 +9,44 @@ excerpt: Multi-cluster Configuration Management
 
 # Projects overview
 
-Federation, in this context, means that a common configuration is pushed out from a central location (Kommander) to all Kubernetes clusters, or a pre-defined subset group under Kommander management. That pre-defined subset group of Kubernetes clusters is called a Project.
+Federation, in this context, means that a central location (Kommander) hosts and pushes common configurations to all Kubernetes clusters, or a pre-defined subset group under Kommander management. That pre-defined subset group of Kubernetes clusters is called a **Project**.
```
Comment (hectorj2f) [inline pages/dkp/kommander/1.4/projects/index.md:12]: I don't need a project to federate stuff using the default workspace.

Comment (makkes) [reply pages/dkp/kommander/1.4/projects/index.md:12]: I'm not sure I understand your suggestion here. Would you mind providing a more accurate wording, please?

Comment (makkes) [reply pages/dkp/kommander/1.4/projects/index.md:12]: Plus, this PR doesn't specifically change the wording around the issue you are (supposedly) addressing. Maybe a subsequent PR should be issued?

Comment (hectorj2f) [reply pages/dkp/kommander/1.4/projects/index.md:12]: I was sharing a detail, I think this statement was omitting. I am not sure if the intention of this Pr is to address all these possible details.

File: pages/dkp/kommander/1.4/projects/index.md [hunk 6]
```diff
@@ -9,40 +9,44 @@ excerpt: Multi-cluster Configuration Management
...
+A Project namespace is a Kommander specific concept that groups all the individual standard Kubernetes namespaces and creates a Project Namespace. Project Namespaces isolate configurations across clusters. Individual standard Kubernetes namespaces are automatically created on all clusters belonging to the project. When creating a new project, you can customize the Kubernetes namespace name that is created.
 
-## Create a Project
+## Create a project
 
-When you create a Project, you must specify a Project Name, a Namespace Name (optional) and a way for Kommander to determine which Kubernetes clusters are part of this project.
+Create a Project by specifying:
+- A Project Name
+- A Namespace Name (optional) 
+- A way for Kommander to determine which Kubernetes clusters are part of this project
```
Comment (hectorj2f) [inline pages/dkp/kommander/1.4/projects/index.md:32]: Should we be more specific and mention `labels` in the UI and a fragment of a YAML when doing it manually via the CLI ? The usage of `a way` is quite abstract to me. 

Note: I know that `a way` is inherited from the old sentence, just trying to find together a better wording.

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/1.4/projects/index.md (hunk 5 above)
Comment (makkes) [reply pages/dkp/kommander/1.4/projects/index.md:12]: No, as the title suggests this PR is mainly about adding details on kubefed placement selectors to the documentation.

Comment (makkes) [reply pages/dkp/kommander/1.4/projects/index.md:12]: I'd be happy to see this page further improved, though, @hectorj2f.

File: pages/dkp/kommander/1.4/projects/index.md (hunk 6 above)
Comment (makkes) [reply pages/dkp/kommander/1.4/projects/index.md:32]: This is explained in more detail two paragraphs down.

File: pages/dkp/kommander/1.4/projects/index.md (hunk 5 above)
Comment (hectorj2f) [reply pages/dkp/kommander/1.4/projects/index.md:12]: Sure, continue improving it 👍🏻

**Pull request merged**